app/hotels/src/map/allHotels/MapView.js

```diff
@@ -64,7 +64,8 @@ export class Map extends React.Component<Props, State> {
     this.markers[selectedIndex] && this.markers[selectedIndex].showCallout();
   };
 
-  componentWillReceiveProps = (nextProps: Props) => {
+  UNSAFE_componentWillReceiveProps = (nextProps: Props) => {
+    // TODO: refactor, not quite sure how to do this in static method
```

Comment:
I think you have to do it the same way like in the `MapScreen`. See: https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html#updating-state-based-on-props

Reply:
Separate PR, please.